src/mod.ts

```diff
@@ -36,6 +36,12 @@ const run = async () => {
 
   options.log(options);
 
+  // Clean output directory
+  if (options.outputDirectory) {
```

Comment:
Might need to `Deno.stat` the output dir first, check if it throws in case we lack permissions for it or it does not exist.

Reply:
Thanks for feedback @filmaj!

In commit [0e10673](https://github.com/slackapi/deno-slack-builder/pull/8/commits/0e1067302eaf5e5a0ca1cd0433abff610eff16b0) I've added a `removeDirectory` function that will gracefully handle removing missing directories.

Looking around a little, I discovered that Deno's [fs.exists implementation is deprecated](https://github.com/denoland/deno_std/blob/0.131.0/fs/exists.ts) because of a race-condition. Instead, they encourage developers to handle the thrown error from `Deno.remove` and inspect the type of error. So that's what I tried to do with our `removeDirectory` function.